__tests__/app.test.js

```diff
@@ -264,13 +264,14 @@ describe("POST /api/articles/:article_id/comments", () => {
 });
 
 describe("PATCH /api/articles/:article_id", () => {
-    xtest("responds with a status code: 200 and sends a single updated article to the client if article_id is valid", () => {
+    test("responds with a status code: 200 and sends a single updated article to the client if article_id is valid", () => {
         const newVotes = {
```

Comment:
this description is what you should be doing, but you've sent -100 votes, so therefore you're not actually receiving back an updated article?